Modules/ITS/src/TrendingTaskITSFhr.cxx

```diff
@@ -0,0 +1,327 @@
+// Copyright CERN and copyright holders of ALICE O2. This software is
+// distributed under the terms of the GNU General Public License v3 (GPL
+// Version 3), copied verbatim in the file "COPYING".
+//
+// See http://alice-o2.web.cern.ch/license for full licensing information.
+//
+// In applying this license CERN does not waive the privileges and immunities
+// granted to it by virtue of its status as an Intergovernmental Organization
+// or submit itself to any jurisdiction.
+
+///
+/// \file    TrendingTaskITSFhr.cxx
+/// \author  Ivan Ravasenga on the structure from Piotr Konopka
+///
+
+#include "ITS/TrendingTaskITSFhr.h"
+#include "../../../Framework/src/RootClassFactory.h"
+#include "QualityControl/DatabaseInterface.h"
+#include "QualityControl/MonitorObject.h"
+#include "QualityControl/QcInfoLogger.h"
+#include "QualityControl/Reductor.h"
+#include "../../Common/include/Common/TH2XlineReductor.h"
```

Comment:
#include "Common/TH2XlineReductor.h" doesn't work? If not, I will have to see why.

Reply:
Sorry I cannot try anymore since I moved already that reductor to ITS module